.github/workflows/ci.yml

```diff
@@ -152,6 +152,20 @@ jobs:
       - name: Build static
         run: yarn build
 
+  backendlistener:
```

Comment:
some thoughts here, maybe it would be good to run those workflows only if files changed.

I believe, the plugin would not be changing that often and we don't need it all the time. Since we run all jobs in parallel  it's not a problem and doesn't affect performance of github actions